src/visit_vtk/full/vtkOBJWriter.C

```diff
@@ -209,7 +209,7 @@ void vtkOBJWriter::WriteData()
         for (cells->InitTraversal(); cells->GetNextCell(npts,indx); )
         {
 #else
-        cells = vtk::TakeSmartPointer(polydata->GetPolys()->NewIterator());
+        auto cells = vtk::TakeSmartPointer(polydata->GetPolys()->NewIterator());
```

Comment:
Isn't this a shadow declaration? I guess this works but it makes me nervous.

Reply:
Shadow declaration?

Reply:
cells is already declared up above as a `vtkCellArray *`.

Reply:
That declaration is bracketed in a preprocessor define that doesn't apply to the line I changed.
And the types cannot be the same, one is a vtkCellArray, the other a vtkCellArrayIterator.

https://github.com/visit-dav/visit/blob/78bf5a1d8d997ee9020c640d1538120d4be4dace/src/visit_vtk/full/vtkOBJWriter.C#L76-L81

https://github.com/visit-dav/visit/blob/78bf5a1d8d997ee9020c640d1538120d4be4dace/src/visit_vtk/full/vtkOBJWriter.C#L207-L216

Reply:
Oh, of course! Not sure why I didn't see the declaration at the top was bracketed.

Everything looks good!